src/util/http.py

```diff
@@ -0,0 +1,136 @@
+from __future__ import annotations
```

Comment:
i'm not sure this is general or useful for most bots enough to include in the template

Reply:
Then they can use it whenever they like.